we-dance/platform
# First Iteration of integration of editor js Feat/69

First Iteration of integration of editor js into our project.
### How to test ###
Go to : https://deploy-preview-174--wedance.netlify.app/demo/texteditor/
You should see the editor js functioning.

Closes #69
